site/src/components/Footer.astro

```diff
@@ -66,15 +66,14 @@ const { class: className } = Astro.props;
   </div>
 
   <p class="text-sm text-center sm:text-left text-dark-40 dark:text-light-40 px-3">
-    © 2010&ndash;{new Date().getFullYear()}, Video.js contributors
+    Video.js is a free and open source HTML5 <a href="/" class="intent:decoration-wavy">video player</a>.
     <br />
-    Video.js is a free and open source HTML5 video player framework
-    <br />
-    Sponsored by <a
+    &copy; 2010&ndash;{new Date().getFullYear()} Video.js contributors &nbsp;|&nbsp; Sponsored by <a
       href="https://www.mux.com?utm_source=videojs&utm_campaign=vjs10"
       class="underline intent:decoration-wavy"
     >
       Mux
     </a>
+    <br />"VIDEO.JS" is a registered trademark of Brightcove Inc.
```

Comment:
To my taste, this flows better as the second line, but that's such a not-big-deal I almost didn't mention it